sdk/python/kfp/_client.py

```diff
@@ -186,8 +190,12 @@ def _extract_pipeline_yaml(self, package_file):
         if len(all_yaml_files) == 0:
           raise ValueError('Invalid package. Missing pipeline yaml file in the package.')
 
-        if len(all_yaml_files) > 1:
-          raise ValueError('Invalid package. Multiple yaml files in the package.')
+        pipeline_file = all_yaml_files[0]
+        if 'pipeline.yaml' in all_yaml_files:
+          pipeline_file = 'pipeline.yaml'
+        else:
+          if len(all_yaml_files) > 1:
+            raise ValueError('Invalid package. Missing pipeline yaml file in the package and no pipeline.yaml file.')
```

Comment:
"Missing pipeline yaml file in the package and no pipeline.yaml file" - seems redundant. It can just be "No pipeline.yaml file in the package"

Reply:
There is a difference. To preserve backwards compatibility, if there is just one yaml file, we load it regardless of the name.

Reply:
Hmm. The wording is strange, yes. I'll fix that.

Reply:
Done